readme.md

```diff
@@ -0,0 +1,178 @@
+<a name="readme-top"></a>
+
+<div align="center">
+   <br/>
+
+  <h3><b>Expense App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the Project ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 About the Project <a name="about-project"></a>
+
+**Expense App** is an transaction tracker app containing data about transactions based on categories. A user can register/login and create transactions to track them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Databases</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Contains data about categories**
+- **Contains data about transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Link to Live demo](https://www.loom.com/share/b6bc3659c096497d8033110ade38c3f6?sid=544571df-58df-4030-a0f0-b49960266c29)
+- [Quick Overview](https://obibaadoma.github.io/Expense-App/)
```

Comment:
- [ ] [REQUIRED] Great effort on deploying the project, but GitHub Pages, used for deployment, isn't ideal for dynamic projects like Ruby on Rails. Consider deploying on Render, Netlify, or Heroku for better compatibility. Include the deployment link in the README. This adjustment ensures your project is showcased in a more suitable environment. Looking forward to the updated deployment.

### Only the README was deployed
https://ibb.co/vq1JW5W